nemo/collections/asr/parts/submodules/conformer_modules.py

```diff
@@ -146,6 +157,16 @@ def forward(
                 cache=cache_last_channel,
                 cache_next=cache_last_channel_next,
             )
+        elif self.self_attention_model == 'rel_pos_local_attn':
```

Comment:
What is the different between 'rel_pos' and 'rel_pos_local_attn' when calling the forward? If there is no difference, let's merge them in one case.

Reply:
There is difference in what is passed to mask, merged into one